core/src/main/java/org/stellar/anchor/sep6/Sep6TransactionUtils.java

```diff
@@ -1,20 +1,26 @@
 package org.stellar.anchor.sep6;
 
+import java.net.MalformedURLException;
+import java.net.URISyntaxException;
 import java.util.ArrayList;
 import java.util.List;
 import org.stellar.anchor.api.sep.sep6.Sep6TransactionResponse;
 import org.stellar.anchor.api.shared.RefundPayment;
 import org.stellar.anchor.api.shared.Refunds;
+import org.stellar.anchor.sep24.MoreInfoUrlConstructor;
 
 public class Sep6TransactionUtils {
 
   /**
    * Converts a SEP-6 database transaction object to a SEP-6 API transaction object.
    *
    * @param txn the SEP-6 database transaction object
+   * @param moreInfoUrlConstructor the more_info_url constructor created from SEP-6 config
    * @return the SEP-6 API transaction object
    */
-  public static Sep6TransactionResponse fromTxn(Sep6Transaction txn) {
+  public static Sep6TransactionResponse fromTxn(
+      Sep6Transaction txn, MoreInfoUrlConstructor moreInfoUrlConstructor)
+      throws MalformedURLException, URISyntaxException {
```

Comment:
Since we never catch and handle the URL exceptions, I think we should catch them and throw a suitable `RuntimeException` instead. Otherwise, you have leaking `throws MalformedURLException, URISyntaxException` to the caller of this method.